repo-agent/pkg/llm/command_expansion_test.go

```diff
@@ -0,0 +1,79 @@
+package llm
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestExpandCommands(t *testing.T) {
+	// Setup temporary .gemini directory
+	tempDir := t.TempDir()
+	geminiDir := filepath.Join(tempDir, ".gemini")
+	commandsDir := filepath.Join(geminiDir, "commands")
+
+	// Create namespace directory
+	nsDir := filepath.Join(commandsDir, "google-internal")
+	if err := os.MkdirAll(nsDir, 0755); err != nil {
+		t.Fatal(err)
+	}
+
+	// Create command file
```

Comment:
The test suite lacks a case for a prompt that contains a command without any arguments. Since the regex requires arguments, this is a key scenario to test to ensure the system either handles it gracefully or that the behavior is explicitly defined.